com.unity.render-pipelines.high-definition/Runtime/Lighting/LightLoop/LightLoopDef.hlsl

```diff
@@ -164,6 +165,21 @@ float4 SampleEnv(LightLoopContext lightLoopContext, int index, float3 texCoord,
         color.rgb = SampleSkyTexture(texCoord, lod, sliceIdx).rgb;
         // Sky isn't pre-expose, so best to clamp to max16 here in case of inf
         color.rgb = ClampToFloat16Max(color.rgb);
+
+#ifdef UNITY_ATMOSPHERIC_SCATTERING_INCLUDED
```

Comment:
this is currently not define anywhere

Reply:
it's defined when the atmospheric scattering file is included, not very clean but it works at least for a draft